docs/running.rst

```diff
@@ -317,6 +317,57 @@ There are several recommended ways to run Toil jobs in the cloud. Of these, runn
 
 On all cloud providers, it is recommended that you run long-running jobs on remote systems under ``screen``. Simply type ``screen`` to open a new ``screen` session. Later, type ``ctrl-a`` and then ``d`` to disconnect from it, and run ``screen -r`` to reconnect to it. Commands running under ``screen`` will continue running even when you are disconnected, allowing you to unplug your laptop and take it home without ending your Toil jobs.
 
+.. _Autoscaling:
+
+Autoscaling
+-----------
+The fastest way to get started runnning Toil in a cloud environment is using
+Toil's autoscaling capabilities to handle node provisioning for us.
+Currently, autoscaling is only supported on the AWS cloud platform with two
+choices of provisioners: Toil's own Docker based provisioner and CGCloud.
+
+The AWS provisioner is included in Toil alongside the [aws] extra and allows
+us to spin up a cluster without any external dependencies using the Toil
+Appliance docker image. To get started, we choose which version of the
```

Comment:
```
using the Toil Appliance, a Docker image that bundles Toil and all its requirements, e.g. Mesos.
```

Or something like that. Once `Toil Appliance` is introduced, I would switch to using `appliance` for short.